src/PlusServer/Commands/vtkPlusCapistranoCommand.h

```diff
@@ -0,0 +1,82 @@
+/*=Plus=header=begin======================================================
+  Program: Plus
+  Copyright (c) Laboratory for Percutaneous Surgery. All rights reserved.
+  See License.txt for details.
+=========================================================Plus=header=end*/
+
+#ifndef __vtkPlusCapistranoCommand_h
+#define __vtkPlusCapistranoCommand_h
+
+#include "vtkPlusServerExport.h"
+
+#include "vtkPlusCommand.h"
+#include "vtkIGSIOTransformRepository.h"
+
+class vtkMatrix4x4;
+
+/*!
+  \class vtkPlusCapistranoCommand
+  \brief This command is for communicating with vtkPlusCapistranoSerialDevices.
+  \ingroup PlusLibPlusServer
+  This command is used for communicating with a Capistrano serial device.
```

Comment:
There seem to be some copy/paste stuff that isn't correct such as describing "Capistrano" as a serial device. Some comments here and the `SetNameToSerial` aren't valid as it is not a serial device. This appears to have been copied over from the GenericSerialDevice class.